### PR TITLE
OLQ: Added LTP and fixed OLQ deployment

### PR DESCRIPTION
**Pull request by mbivolan**

### CODE REVIEW

File: Testscripts/Linux/Linux-Test-Project-Tests.sh [hunk 1]
```diff
@@ -95,39 +93,51 @@ cd "$TOP_SRCDIR"
...
+    LogMsg "Installing LTP..."
+    make -j "$proc_count" install 2>/dev/null
+    if [ $? -ne 0 ]; then
+        LogMsg "Error: Failed to install LTP!"
+        SetTestStateFailed
+        exit 0
+    fi
+else
+    LogMsg "Download ltp package from: $LTP_PACKAGE_URL"
+    curl $LTP_PACKAGE_URL --output "ltp.rpm"
```
Comment (lizzha) [inline Testscripts/Linux/Linux-Test-Project-Tests.sh]: Should you check whether it's an RPM? Will there be deb package provided?

File: Testscripts/Linux/Linux-Test-Project-Tests.sh [hunk 2]
```diff
@@ -82,8 +82,6 @@ case $DISTRO in
         ;;
     *)
         LogMsg "Unknown Distro"
```
Comment (chvalean) [inline Testscripts/Linux/Linux-Test-Project-Tests.sh]: "Unknown Distro, continuing to try for RPM installation" maybe?

File: Testscripts/Linux/Linux-Test-Project-Tests.sh [hunk 3]
```diff
@@ -95,39 +93,51 @@ cd "$TOP_SRCDIR"
...
+        LogMsg "Error: Failed to install LTP!"
+        SetTestStateFailed
+        exit 0
+    fi
+else
+    LogMsg "Download ltp package from: $LTP_PACKAGE_URL"
+    curl $LTP_PACKAGE_URL --output "ltp.rpm"
+    rpm --nodeps -ivh ltp.rpm
+    if [ $? -ne 0 ]; then
+        LogMsg "Error: Failed to install LTP!"
```
Comment (chvalean) [inline Testscripts/Linux/Linux-Test-Project-Tests.sh]: LTP RPM

File: Testscripts/Linux/Linux-Test-Project-Tests.sh (hunk 1 above)
Comment (mbivolan) [reply Testscripts/Linux/Linux-Test-Project-Tests.sh]: We will only user rpms.

File: Testscripts/Linux/Linux-Test-Project-Tests.sh [hunk 4]
```diff
@@ -95,39 +93,51 @@ cd "$TOP_SRCDIR"
...
 # You can run the ltp test with any tag using LISAv2's Custom Parameters feature.
-LogMsg "Cloning LTP"
-git clone https://github.com/linux-test-project/ltp.git
-TOP_SRCDIR="${HOME}/src/ltp"
 
-cd "$TOP_SRCDIR"
-if [[ "$ltp_version_git_tag" != "" || "$ltp_version_git_tag" != "master" ]]; then
-    git checkout tags/"$ltp_version_git_tag"
-fi
+if [[ $LTP_PACKAGE_URL == "" ]];then
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/Linux-Test-Project-Tests.sh]: This change should be verified in Hyper-V and Azure and the result shows no-broken.

File: XML/Other/ReplaceableTestParameters.xml [hunk 5]
```diff
@@ -361,4 +361,8 @@ Scenario 2 : You need to run all the performance tests quickly.
         <!-- Reference: https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html -->
         <ReplaceWith>10.0.130-1</ReplaceWith>
     </Parameter>
+    <Parameter>
+        <ReplaceThis>LTP_BUILD_URL</ReplaceThis>
+        <ReplaceWith></ReplaceWith>
```
Comment (juhlee-microsoft) [inline XML/Other/ReplaceableTestParameters.xml]: Empty value?

Comment (mbivolan) [reply XML/Other/ReplaceableTestParameters.xml]: Its a secret, needs to be specified at runtime, always.

File: Testscripts/Linux/Linux-Test-Project-Tests.sh (hunk 4 above)
Comment (mbivolan) [reply Testscripts/Linux/Linux-Test-Project-Tests.sh]: Its tested

**Pull request merged**